flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV120.java

```diff
@@ -148,6 +151,9 @@ public CatalogColumnStatisticsDataDate toFlinkDateColStats(ColumnStatisticsData
 
 	@Override
 	public Optional<FunctionInfo> getBuiltInFunctionInfo(String name) {
+		if (isCatalogFunctionName(name)) {
+			return Optional.empty();
+		}
 		Optional<FunctionInfo> functionInfo = getFunctionInfo(name);
 
 		if (functionInfo.isPresent() && isBuiltInFunctionInfo(functionInfo.get())) {
```

Comment:
isn't built in functions filtered here already?

Reply:
Since this is a public API, I'd rather not to assume the name has been filtered. I'll do some refactor to avoid duplicated code

Reply:
I have same question, why not filter by `isBuiltInFunctionInfo`?